tamr_toolbox/data_io/dataframe.py

```diff
@@ -372,7 +402,14 @@ def validate(
         _check_nonnull_columns(df_profile, require_nonnull_columns=require_nonnull_columns).details
     )
 
-    # convert to dict for printing/return
+    if custom_checks is not None:
+        for custom_check in custom_checks:
+            failed_checks_dict.update(
+                _check_custom(
+                    df, check_function=custom_check[0], columns_to_check=custom_check[1],
+                ).details
+            )
```

Comment:
If the above default value for `custom_checks` of `tuple()` (or `()`) is made, this can just be
```suggestion
    for custom_check in custom_checks:
        failed_checks_dict.update(
            _check_custom(
                df, check_function=custom_check[0], columns_to_check=custom_check[1],
            ).details
        )
```